### DISCUSSION TITLE
How to use postgres-operator helm chart Timescaledb  with community edition

### DISCUSSION THREAD

**Discussion by abbabe** [zalando/postgres-operator#2792] [General]
I installed postgres-operator helm chart, I also added timescaledb extension, but the version is Apache 2 Edition.  How can I set timescaledb Community Edition.

**Comment by ennc0d3** [comment]
You have to build the spilo image with custom build args,			
                                 --build-arg TIMESCALEDB_APACHE_ONLY=false \
				--build-arg TIMESCALEDB_TOOLKIT=true\
